intro.md

```diff
@@ -1,24 +0,0 @@
-# Introducing dcrpool.
```

Comment:
Did you intend to delete this file?

Reply:
yes, it has served its purpose and is no longer needed.